packages/contracts/contracts/interfaces/IExternalRewardsDistributor.sol

```diff
@@ -11,29 +11,18 @@ interface IExternalRewardsDistributor {
     /// @param newRoot The new merkle's tree root.
     event RootUpdated(bytes32 newRoot);
 
-    /// @notice Emitted when MORPHO tokens are withdrawn.
-    /// @param to The address of the recipient.
-    /// @param amount The amount of MORPHO tokens withdrawn.
-    event MorphoWithdrawn(address to, uint256 amount);
-
     /// @notice Emitted when an account claims rewards.
     /// @param account The address of the claimer.
     /// @param amount The amount of rewards claimed.
     event RewardsClaimed(address account, uint256 amount);
 
-    /// ERRORS ///
-
-    /// @notice Thrown when the proof is invalid or expired.
-    error ProofInvalidOrExpired();
-
-    /// @notice Thrown when the claimer has already claimed the rewards.
-    error AlreadyClaimed();
-
     event RewardConfigured(address indexed aToken, address indexed staking, uint256 initialAmount);
     event StakingRemoved(address indexed aToken);
-    event UserDeposited(address indexed user, address indexed underlying, uint64 indexed trancheId, uint256 amount);
-    event UserWithdraw(address indexed user, address indexed underlying, uint64 indexed trancheId, uint256 amount);
-    event UserTransfer(address indexed user, address indexed underlying, uint64 indexed trancheId, uint256 amount, bool sender);
+    event UserDeposited(address indexed user, address indexed aToken, uint256 amount);
+    event UserWithdraw(address indexed user, address indexed aToken, uint256 amount);
+    event UserTransfer(address indexed user, address indexed aToken, uint256 amount, bool sender);
+
+    event HarvestedReward(address indexed stakingContract, address[] rewardTokens, uint256[] earned);
```

Comment:
thi should be only HarvestedReward(address stakingContract), we dont need to track rewardToken and earned, everything is parseable through ERC20 Transfer events in off chain external reward processor